Task_1/Task_1/src/components/Input_form/form.jsx

```diff
@@ -0,0 +1,66 @@
+import { useState } from "react";
+
+const Form = (props) => {
+  const [enterTask, setEnterTask] = useState("");
+  const [enterDate, setEnterDate] = useState("");
```

Comment:
Date should not be string, instread use null initially and update date as new Date not by converting to string.

Reply:
Implemented